scripts/build/gen_relocate_app.py

```diff
@@ -513,27 +519,31 @@ def parse_input_string(line):
         mem_region = mem_region.rstrip()
         phdr, rest = rest.split(':', 1)
 
+    flag_list, rest = rest.split(':', 1)
+
     # Split lists by semicolons, in part to support generator expressions
-    flag_list, file_list = (lst.split(';') for lst in rest.split(':', 1))
+    file_list, symbol_filter = (lst.split(';') for lst in rest.split(',', 1))
```

Comment:
Why is `symbol_filter` split by `;` and returned as a list?

Reply:
it's because that allows the use of generator expressions from CMake which could form a `;`-separated list.
Used already today for file lists, and considered it would make sense to support the same for the symbol filter, but can take it out if prefered.

Can be added when there is an actual need.

Reply:
Okay. Currently, only the first regex seems to be used. If we were to allow multiple, we'd need to decide whether to treat as AND- or OR-clauses, but all of this can be described in a single regex. A CMake genex can also be joined into one.

Reply:
Removed.